src/styles/common.scss

```
@@ -1,3 +1,33 @@
-*{
+$bg-color: #FAFAFA;
+$theme-color: #FFFFFF;
+$login-btn-color: #0095F6;
+@mixin border {
+  border: 1px solid #DBDBDB;
+}
+@mixin flex-align-center {
+  display: flex;
+  align-items: center;
+}
+@mixin flex-justify-center {
+  display: flex;
+  justify-content: center;
+}
+@mixin flex-justify-between {
+  display: flex;
+  justify-content: space-between;
+}
```

```suggestion
@mixin center {
  display: flex;
  align-items: center;
  justify-content: center;  
}
```
이런식으로도 한 번 사용해보세요~@!